src/main/java/edu/umich/its/cpm/MigrationTaskService.java

```diff
@@ -352,9 +386,10 @@ public void setMigrationEndTimeAndStatus(String migrationId,
 						String zipFileStatus = zipFiles(type, httpContext,
 								zipFileName, title, webLinkUrl, contentAccessUrl,
```

Comment:
This `zipFileStatus`  method will return for example "this is successful something something" the function call `finalReportObjBuilderForMailZipMigration` is actually looking for Status = OK/PARTIAL/ERROR. This is will break the report object of the Mail zip Migration